edgepi-digital-out.py

```diff
@@ -6,27 +6,37 @@
 _logger=logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
-# arv[0] is this file's name, argv[1] should be numeric command
-if len(sys.argv) > 1:
+# create new DOUT instance
+try:
+    digital_output = EdgePiDigitalOutput()
+except ModuleNotFoundError as e:
+    _logger.error(f'Failed to load EdgePi modules: {e}')
+    sys.exit(0)
+
+# node channel property to SDK enum
+channelType_to_enum = {"dout1": GpioPins.DOUT1, "dout2": GpioPins.DOUT2, "dout3": GpioPins.DOUT3, "dout4": GpioPins.DOUT4,
+"dout5": GpioPins.DOUT5, "dout6":GpioPins.DOUT6, "dout7":GpioPins.DOUT7, "dout8": GpioPins.DOUT8}
```

Comment:
you don't need this dictionary if you follow my other comment about using the enum name directly